docs/en/operations/table_engines/mergetree.md

```diff
@@ -70,11 +70,13 @@ For a description of parameters, see the [CREATE query description](../../query_
 
     If a sampling expression is used, the primary key must contain it. Example: `SAMPLE BY intHash32(UserID) ORDER BY (CounterID, EventDate, intHash32(UserID))`.
 
-- `TTL` — An expression for setting storage time for rows.
+- `TTL` — A list of rules specifying durations of storage of rows and defining logic of automatic moving parts between disks and volumes.
```

Comment:
```suggestion
- `TTL` — A list of rules specifying storage duration of rows and defining logic of automatic parts movement between disks and volumes.
```
also probably needs a link to docs on how to specify disks/volumes in configuration